src/contract.rs

```diff
@@ -122,9 +171,10 @@ fn create_pool(
     token1: String,
     chain0_init_depositor: String,
     chain1_init_depositor: String,
+    fee: u16,
 ) -> Result<Response<PalomaMsg>, ContractError> {
     assert!(chain0_id < chain1_id);
```

Comment:
If we don't asset this inequality, is it going to create problems somewhere? This design requires the two chains to be different, which is not necessary.